public/js/components/pay-method-icon.jsx

```diff
@@ -0,0 +1,37 @@
+import cx from 'classnames';
+import React, { Component, PropTypes } from 'react';
+
+// Just a convenience mapping for payMethods from payMethod-validator
+// to shorted classes used in CSS.
+const payMethodTypeMap = {
+  'american-express': 'amex',
+  'diners-club': 'diners',
+  'master-card': 'mastercard',
+};
```

Comment:
This specific is just a mapping for being able to handle cardTypes handed to us by the card validator which have slight variants on the names for cards we use internally. 

A paymethod that's an account will just be handled slightly differently it's still essentially all the same thing.